.github/workflows/push.yml

```diff
@@ -55,13 +55,13 @@ jobs:
 
       - uses: atymic/sitemap-ping-action@master
         with:
-          sitemap-url: https://bjerk.io/sitemap.xml
+          sitemap-url: https://bjerk.io/sitemap/sitemap-index.xml
```

Comment:
Why has this changed? Do we know?

Wondering if need to change some references elsewhere too. Might be some Google stuff that needs an update. 🤷‍♂️

Reply:
Could it be a `gatsby-plugin-sitemap` update that cause this?